test/integration/test/sinker_test.go

```diff
@@ -302,7 +302,7 @@ func TestDeletePod(t *testing.T) {
 				return scheduled_for_deletion, nil
 			})
 			// Check for the error of `List` call.
-			if err != nil && !wait.Interrupted(err) {
+			if err != nil {
```

Comment:
Sorry I am late here but with L288 change this line now re-checks the `err` from L272 which does not seem correct. This change essentially reverts 6c290be6d3b18042be29e7285372f98a70a40026 but not its part makes `err` not a closure?

@Prucek @smg247 PTAL

Reply:
Yeah, I missed that we were now shadowing `err`. I will fix that in a quick followup.